### PR TITLE
Add an example of using popper.js to show a tooltip

### PR DESCRIPTION
**Pull request by cnwangjie**

### CODE REVIEW

File: examples/charts/experimental.htm [hunk 1]
```diff
@@ -626,6 +626,94 @@
...
+</code></pre>
+
+<pre><code class='javascript'>const tooltipEl = d3.select('.tooltip').node();
+const tooltip = new Popper(document.documentElement, tooltipEl, { placement: 'top' });
+
+d3.json('data/fake_users1.json', function(data) {
+    data = MG.convert.date(data, 'date');
+    MG.data_graphic({
+        title: "Tooltip on line chart",
+        description: "This is an example to use popper.js to show a tooltip on line chart.",
```
Comment (wlach) [inline examples/charts/experimental.htm]: Change this (here and below, too bad we duplicate the code in two places for these examples) to:

`Using popper.js and the mouseover callback, you can easily create a tooltip when the user hovers over a particular datapoint.`

File: examples/charts/experimental.htm [hunk 2]
```diff
@@ -626,6 +626,94 @@
...
+            tooltip.reference = d3.select(`#tooltip-line-chart .mg-line-rollover-circle`).node();
+            tooltip.update();
+        }
+    });
+});
+
+d3.json('data/points1.json', function(data) {
+    MG.data_graphic({
+        title: "Tooltip on point chart",
+        description: "This is an example to use popper.js to show a tooltip on point chart.",
```
Comment (wlach) [inline examples/charts/experimental.htm]: Change text to: `This technique also works for point charts`

File: examples/charts/experimental.htm [hunk 3]
```diff
@@ -626,6 +626,94 @@
...
+        target: '#tooltip-line-chart',
+        x_accessor: 'date',
+        y_accessor: 'value',
+        show_rollover_text: false,
+        mouseover: (d, i) => {
+            tooltipEl.style.display = 'block';
+            tooltipEl.innerText = `date: ${d3.timeFormat('%b %e, %Y')(d.date)}\nvalue: ${d.value}`;
+            tooltip.reference = d3.select(`#tooltip-line-chart .mg-line-rollover-circle`).node();
+            tooltip.update();
+        }
```
Comment (wlach) [inline examples/charts/experimental.htm]: Can we use the mouseout callback to make the tooltip disappear when the user hovers away from the graph?

**Comment by cnwangjie** [comment]
@wlach Ok, I've submitted the changes. I found that if just hide tooltip on mouse out of point, the tooltip will flash. So I also bind an event on the tooltip element to avoid this.

**Pull request merged**